tasks/recover/register_template.yml

```
@@ -0,0 +1,23 @@
+- block:
+  - name: Register unregistered Template
+    ovirt_templates:
+        state: registered
+        storage_domain: "{{ storage.name }}"
+        id: "{{ unreg_template.id }}"
+        allow_partial_import: "{{ dr_partial_import }}"
+        auth: "{{ ovirt_auth }}"
+        cluster_mappings: "{{ dr_cluster_map }}"
+        domain_mappings: "{{ dr_domain_map }}"
+        vnic_profile_mappings:  "{{ dr_network_map }}"
+        role_mappings: "{{ dr_role_map }}"
+    register: template_register_result
```

Should you ignore errors here? otherwise it will fail and stop , no?